lib/packages/shared-utils/legacy-event-type.ts

```diff
@@ -0,0 +1,56 @@
+import { Action } from "shared-types";
+
+// Resolve the action type based on the GSI1pk
+export const getLegacyEventType = (GSI1pk: string) => {
+  if (!GSI1pk) {
+    return undefined;
+  }
+  const submitType = GSI1pk?.split("OneMAC#submit")?.[1] || "";
```

Comment:
```suggestion
  const submitType = GSI1pk.split("OneMAC#submit")?.[1] || "";
```

You can remove if that variable is defined